src/Focusable.ts

```diff
@@ -202,12 +218,15 @@ export class FocusableAPI implements Types.FocusableAPI {
         const elements: HTMLElement[] = [];
 
         let { acceptCondition } = options;
+        let hasCustomCondition = true;
```

Comment:
```suggestion
        const hasCustomCondition = !!acceptCondition;
```

This can be constant, there's no reason for this to be assigned later